Rocket.Chat/Resources/pt-BR.lproj/Localizable.strings

```diff
@@ -226,12 +226,16 @@
 
 // Members List
 "chat.members.list.title" = "Membros";
+"chat.member.unblock.title" = "Desbloquear";
+"chat.member.open.profile.title" = "Perfil aberto";
```

Comment:
Abrir perfil

Reply:
Thanks! I've just used translator for these texts :)